### PR TITLE
added bottom padding to each page

### PR DESCRIPTION
**Pull request by radhikabhatt26**

### CODE REVIEW

**Comment by arowla** [comment]
Hi @radhikabhatt26 , the original fix for this didn't require adding something to each page-- and I see that my original fix in the CSS is now commented out as a result of the commit below. Is there any way we can get back to a more universal fix?

https://github.com/radhikabhatt26/ITA_Principal_Travel/commit/dcda4bcc08d83eb0f1f250506a5964db7d406666#diff-c7c46c788a7fb0f0e5bbcf03d476b5af

Under this fix, won't every new page need a special class added just to not overlap the footer? cc@nkalbasi

**Comment by radhikabhatt26** [comment]
@arowla I reverted back to the universal fix and fixed the white space issue.

File: principal_travel/templates/travel/event_detail.html [hunk 1]
```diff
@@ -19,7 +19,8 @@
 </div>
   {% include 'travel/_authortime.html'%}
 <br>
+<div class="end-button">
```
Comment (arowla) [inline principal_travel/templates/travel/event_detail.html]: Is `end-button` still relevant?

**Comment by arowla** [comment]
Thank you!

**Pull request merged**